cards.js

```diff
@@ -0,0 +1,44 @@
+
+
+var cardNum = ["Ace","2","3","4","5","6","7","8","9","10","Jack","Queen","King"];	// Array of Card Number
+var cardShape = ["Clubs","Spades","Hearts","Diamonds"];								//Array of Card Shape
+var cardDist=[];						//Array of card deck
+var k=0;
+
+for(var i=0;i<cardNum.length;i++){
+	for(var j=0;j<cardShape.length;j++){
+		cardDist[k]= cardNum[i]+" of "+cardShape[j];		

```

Comment:
make card and object and push object rather than string,
having card as a object will later help you comparing strength of cards